sdk/core/azure-core/inc/azure/core/base64.hpp

```diff
@@ -41,6 +41,14 @@ namespace Azure { namespace Core {
      */
     static std::string Base64Encode(const std::vector<uint8_t>& data);
 
+    /**
+     * @brief Encodes a string using Base64 encoding.
+     *
+     * @param data The input string that contains data to be encoded.
+     * @return The Base64 encoded contents of the vector.
+     */
+    static std::string Base64Encode(const std::string& data);
```

Comment:
It never went away. I just had no public reasons to expose it. But proxy authn support requires base64 encoding a string, so I added a specialization for `Base64Encode` which takes a string to avoid the extra memory copy.